app/coffee/modules/backlog/main.coffee

```diff
@@ -712,6 +712,11 @@ class BacklogController extends mixOf(taiga.Controller, taiga.PageMixin, taiga.F
         @.filtersReloadContent()
         @.generateFilters('null')
 
+    clearFiltersBacklog: () ->
+        @.replaceAllFilters({})        
+        @.filtersReloadContent()
+        @.generateFilters()
```

Comment:
other commands pass `'null'` as parameter?

Reply:
Yes, I also noticed it. In the project, they either pass nothing or null. It wont make much difference as the only place, this parameter is used is in [here](https://github.com/taigaio/taiga-front/blob/91bea87500cada61fe866b13369133b9e054504d/app/coffee/modules/controllerMixins.coffee#L242-L243). `undefined` and `null` will probably result in the same.

Reply:
for consistency I would also pass `'null'` here. Also `'null'` is not `null`

```suggestion
        @.generateFilters('null')
```

Reply:
Ah, richtig, hab das als `null` und nicht als `'null'` gelesen 😅 Ist hinzugefügt